AmericanExpress/src/main/java/com/braintreepayments/api/americanexpress/AmericanExpressAnalytics.kt

```diff
@@ -1,4 +1,4 @@
-package com.braintreepayments.api
+package com.braintreepayments.api.americanexpress
```

Comment:
This is going to be a _super basic_ question, but what is the advantage to moving to appending `americanexpress`? 😅

Reply:
I also assume we will make this change in all of the modules and not just amex or no? Neither of these are blockers just for my own knowledge.

Reply:
Ah nice! That is super helpful. 🙏

Reply:
> I also assume we will make this change in all of the modules and not just amex or no?

Yep! We'll be updating the package name for all modules. I spit it up to make the PRs more digestible.